src/urllib3/util/ssl_.py

```diff
@@ -246,24 +276,62 @@ def create_urllib3_context(
     if (cert_reqs == ssl.CERT_REQUIRED or sys.version_info >= (3, 7, 4)) and getattr(
         context, "post_handshake_auth", None
     ) is not None:
-        context.post_handshake_auth = True
+        context.post_handshake_auth = True  # type: ignore
```

Comment:
As far as I understand this works for versions 3.7.4+, but typeshed includes it only for 3.8+ (SSLContext)